src/app/common/components/bg-badgecard.ts

```diff
@@ -2,54 +2,80 @@ import { Component, EventEmitter, Input, Output } from '@angular/core';
 
 @Component({
 	selector: 'bg-badgecard',
-	host: { class: 'badgecard' },
+	host: {
+		class: 'tw-rounded-[10px] tw-bg-white tw-border-purple tw-border-solid tw-border tw-relative tw-p-[calc(var(--gridspacing)*3)] tw-block tw-overflow-hidden oeb-badge-card',
+	},
 	template: `
-		<div class="badgecard-x-status badgestatus badgestatus-{{ mostRelevantStatus }}" *ngIf="mostRelevantStatus">
+		<div
+			class="tw-absolute tw-top-0 tw-left-0 tw-bg-purple tw-text-white tw-px-2 tw-py-1"
+			*ngIf="mostRelevantStatus"
+		>
 			{{ mostRelevantStatus }}
 		</div>
 
-		<div class="badgecard-x-body">
-			<div class="badgecard-x-image">
-				<img
-					class="badgeimage badgeimage-{{ mostRelevantStatus }}"
-					[loaded-src]="badgeImage"
-					[loading-src]="badgeLoadingImageUrl"
-					[error-src]="badgeFailedImageUrl"
-					width="80"
-				/>
-			</div>
-			<a
-				*ngIf="badgeSlug"
-				class="badgecard-x-title u-text-breakword"
-				[routerLink]="['../earned-badge', badgeSlug]"
-				>{{ badgeTitle }}</a
-			>
-			<a *ngIf="publicUrl" class="badgecard-x-title" [href]="publicUrl">{{ badgeTitle }}</a>
-			<a
-				*ngIf="issuerSlug; else noIssuerSlug"
-				class="badgecard-x-issuer"
-				[routerLink]="['../../public/issuers', issuerSlug]"
-				>{{ issuerTitle }}</a
-			>
-			<ng-template #noIssuerSlug>
-				<div class="badgecard-x-issuer">{{ issuerTitle }}</div>
-			</ng-template>
-			<p class="badgecard-x-desc" [truncatedText]="badgeDescription" [maxLength]="100"></p>
-		</div>
-		<div class="badgecard-x-footer">
-			<div class="badgecard-x-date">
-				<span *ngIf="badgeClass"> Verfügbar seit: </span>
-				<time [date]="badgeIssueDate" format="mediumDate"></time>
+		<div class="tw-flex tw-items-center">
+			<img
+				class="badgeimage badgeimage-{{ mostRelevantStatus }}"
+				[loaded-src]="badgeImage"
+				[loading-src]="badgeLoadingImageUrl"
+				[error-src]="badgeFailedImageUrl"
+				width="80"
+			/>
+			<div class="tw-flex tw-flex-col tw-flex-wrap tw-pl-4">
+				<a
+					*ngIf="badgeSlug && !publicUrl"
+					class="tw-font-bold"
+					[routerLink]="['../earned-badge', badgeSlug]"
+					hlmP
+					size="sm"
+					>{{ badgeTitle }}</a>
+					 
+				<a *ngIf="publicUrl"
+					class="tw-font-bold"
+					hlmP
+					size="sm" [href]="publicUrl">{{ badgeTitle }}</a>
+
+				<div class="tw-pt-2 tw-flex tw-flex-col tw-flex-wrap">
+					<a
+						hlmP
+						size="sm"
+						variant="light"
+						*ngIf="issuerSlug; else noIssuerSlug"
+						class="badgecard-x-issuer"
+						[routerLink]="['../../public/issuers', issuerSlug]"
```

Comment:
I think it would be more consistent if this also navigated to `issuer/issuer` page, if the user is a member of the institution (similar to the [catalog](https://github.com/mint-o-badges/badgr-ui/blob/main/src/app/catalog/components/issuer-catalog/issuer-catalog.component.ts#L426)).

Reply:
Not sure about this one. I will check the designs again

Reply:
I asked sven how to handle this. i think its a bigger topic, so i would leave it as it is and make the changes later.